pkg/clusters/addons/loadimage/addon.go

```diff
@@ -0,0 +1,101 @@
+package loadimage
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os/exec"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters"
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters/types/kind"
+)
+
+// -----------------------------------------------------------------------------
+// CertManager Addon - Builder
+// -----------------------------------------------------------------------------
+
+type Builder struct {
+	image string
+}
+
+func NewBuilder() *Builder {
+	return &Builder{}
+}
+
+func (b *Builder) WithImage(image string) *Builder {
+	b.image = image
+	return b
+}
+
+func (b *Builder) Build() *Addon {
+	return &Addon{
+		image:  b.image,
+		loaded: false,
+	}
+}
```

Comment:
for consistency with other addons, the builder implementation should probably live in `builder.go`

Reply:
361f92f

Reply:
361f92f